src/components/WebexMeeting/WebexMeeting.jsx

```diff
@@ -39,24 +45,30 @@ export default function WebexMeeting({className, meetingID, style}) {
   } else {
     meetingDisplay = (
       <>
-        {isActive
-          ? <WebexInMeeting meetingID={ID} />
-          : <WebexInterstitialMeeting meetingID={ID} />}
+        <div className="body">
+          {isActive
+            ? <WebexInMeeting meetingID={ID} />
+            : <WebexInterstitialMeeting meetingID={ID} />}
+          {showSettings && (
```

Comment:
settings don't need to belong to .body, please put them at the end, just after the control bar